auacm/app/static/js/controllers/TeamController.js

```diff
@@ -80,8 +80,7 @@ app.controller('TeamController', ['$scope', '$http', '$location', '$window',
         });
     };
 
-    var socket = io.connect('http://' + $window.location.host + '/register');
-    socket.on('connect', function() {});
+    var socket = new Socket('ws://' + $window.location.host + '/websocket');
```

Comment:
That's not a bad idea. I can't think of any negative consequences. I'll do that.